aiida_pseudo/groups/mixins/cutoffs.py

```diff
@@ -299,6 +303,9 @@ def get_recommended_cutoffs(self, *, elements=None, structure=None, stringency=N
 
         for element in symbols:
 
+            if element not in cutoffs:
+                raise ValueError(f'family does not contain a pseudo for element `{element}`.')
```

Comment:
can you add this to the docstring as a potential raise please

Reply:
Of course, silly me!